scripts/setup/postgres-init-db

```diff
@@ -2,32 +2,74 @@
 set -e
 set -x
 
+# A bit of a helper variable, default is voyager
+DEPLOYMENT_TYPE="${DEPLOYMENT_TYPE:-voyager}"
+# What user should we use for connecting to the database
+POSTGRES_USER="${POSTGRES_USER:-postgres}"
+
 # Shut down all services to ensure a quiescent state.
-supervisorctl stop all
+# Stop supervisord only if the socket exists, prevents problems with docker
+if [ -e "/var/run/supervisor.sock" ]; then
+    supervisorctl stop all
+fi
```

Comment:
While building the image, supervisor won't be started. That causes `supervisorctl stop all` to return != 0 and exit the script with an error, this prevent this from happening.